docs/serving/samples/hello-world/helloworld-go/Dockerfile

```diff
@@ -1,28 +1,30 @@
-# Use the official Golang image to create a build artifact.
+# Use the offical golang image to create a binary.
```

Comment:
We may want to keep Golang capitalized for consistency. It doesn't show up a ton but it looks like other locations have it Golang as well.